tests/tqueue_int/tqueue_int.c

```diff
@@ -611,17 +613,33 @@ static void TQUEUE_chaos_knight_test_with_THANDLE_template(uint32_t queue_size,
 // It also uses a condition function for popping which randomly rejects pops
 TEST_FUNCTION(TQUEUE_chaos_knight_test_with_THANDLE)
 {
-    TQUEUE_chaos_knight_test_with_THANDLE_template(16, N_THREADS, N_THREADS, true);
+    TQUEUE_chaos_knight_test_with_THANDLE_template(16, 16, N_THREADS, N_THREADS, true);
 }
 
 TEST_FUNCTION(MU_C3(TQUEUE_chaos_knight_test_with_THANDLE_queue_size_16_and_1_pusher_and_, N_THREADS, _poppers))
 {
-    TQUEUE_chaos_knight_test_with_THANDLE_template(16, 1, N_THREADS, true);
+    TQUEUE_chaos_knight_test_with_THANDLE_template(16, 16, 1, N_THREADS, true);
 }
 
 TEST_FUNCTION(MU_C3(TQUEUE_chaos_knight_test_with_THANDLE_queue_size_16_and_, N_THREADS, _pushers_and_1_popper))
 {
-    TQUEUE_chaos_knight_test_with_THANDLE_template(16, N_THREADS, 1, true);
+    TQUEUE_chaos_knight_test_with_THANDLE_template(16, 16, N_THREADS, 1, true);
+}
+
+// And now the same with queues that grow
+TEST_FUNCTION(TQUEUE_chaos_knight_test_with_THANDLE_grow_queue)
```

Comment:
>_grow_queue [](http://example.com/codeflow?start=51&length=11)

nit: the next 2 tests don't say anything about "grow"ing /poetry/consider

Reply:
Uhm, they do, but maybe it was not that visible since it is as a suffix to the test name

Reply:
yup, I just missed it haha I guess I expected it in a different order, nevermind